src/adapters/analytics/AnalyticsAdapter.js

```diff
@@ -93,6 +94,10 @@ export default function AnalyticsAdapter({ url, analyticsType, global, handler }
     events.on(BID_TIMEOUT, args => this.enqueue({ eventType: BID_TIMEOUT, args }));
     events.on(BID_WON, args => this.enqueue({ eventType: BID_WON, args }));
     events.on(BID_ADJUSTMENT, args => this.enqueue({ eventType: BID_ADJUSTMENT, args }));
+    events.on(AUCTION_INIT, args => {
+      args.memberId = config.options.memberId;  // from enableAnaltyics config object
```

Comment:
Instead of passing a specific param, shouldn't we send all the `options` (since they will vary)?